recipes/large-image-source-gdal/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "large-image-source-gdal" %}
+{% set version = "1.8.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6aa99e60c4c5de5dfb1931255b1d503d2836bacdb37af2216cc37d605b4ef3a1
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - large-image ={{ version }}
```

Comment:
@conda-forge/core, I'm curious if using `={{ version }}` is an acceptable way to make sure all of these `large-image*` packages are always installed with matching versions?

Reply:
This way of doing things appears fine.

A single `=` is equivalent to the pypi `~=` AFAIK and so `=1.0` will match 1.0.0, 1.0.1, etc. If this is what you want, then great! Otherwise you might consider exactly equality `==` or a more liberal range using the jinja2 pinning macros (https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#extra-jinja2-functions, see pin_compatible).

See docs here: https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/pkg-specs.html#examples for info on version operators.

Reply:
Ah, thanks for the tip on `~=`! I actaully think I want `==` here, so I will adjust these